### ISSUE TITLE
Linkerd dashboard is blank

### ISSUE THREAD

**Issue by guimap** [linkerd/linkerd2#5845]
### What is the issue?
When i try access linkerd dashboard, it opens a blank screen

### How can it be reproduced?

When run `linkerd dashboard` into terminal

### Logs, error output, etc

logs from `linkerd-web`
```txt
time="2021-03-01T15:01:12Z" level=info msg="running version stable-2.9.4"
time="2021-03-01T15:01:12Z" level=info msg="starting admin server on :9994"
time="2021-03-01T15:01:12Z" level=info msg="starting HTTP server on :8084"
time="2021-03-01T15:02:26Z" level=error msg="HTTP error, status Code [503] (unexpected API response)"
time="2021-03-01T15:02:26Z" level=error msg="template: app.tmpl.html:3:33: executing \"content\" at <.Data.ReleaseVersion>: nil pointer evaluating *public.VersionInfo.ReleaseVersion"

```

#### `linkerd check` output

```text
kubernetes-api
--------------
√ can initialize the client
√ can query the Kubernetes API

kubernetes-version
------------------
√ is running the minimum Kubernetes API version
√ is running the minimum kubectl version

linkerd-existence
-----------------
√ 'linkerd-config' config map exists
√ heartbeat ServiceAccount exist
√ control plane replica sets are ready
√ no unschedulable pods
√ controller pod is running
 √ can initialize the client
√ can query the control plane API

linkerd-config
--------------
√ control plane Namespace exists
√ control plane ClusterRoles exist
√ control plane ClusterRoleBindings exist
√ control plane ServiceAccounts exist
√ control plane CustomResourceDefinitions exist
√ control plane MutatingWebhookConfigurations exist
√ control plane ValidatingWebhookConfigurations exist
√ control plane PodSecurityPolicies exist

linkerd-identity
----------------
√ certificate config is valid
√ trust anchors are using supported crypto algorithm
√ trust anchors are within their validity period
√ trust anchors are valid for at least 60 days
√ issuer cert is using supported crypto algorithm
√ issuer cert is within its validity period
√ issuer cert is valid for at least 60 days
√ issuer cert is issued by the trust anchor

linkerd-webhooks-and-apisvc-tls
-------------------------------
√ tap API server has valid cert
√ tap API server cert is valid for at least 60 days
√ proxy-injector webhook has valid cert
√ proxy-injector cert is valid for at least 60 days
√ sp-validator webhook has valid cert
√ sp-validator cert is valid for at least 60 days

linkerd-api
-----------
√ control plane pods are ready
√ control plane self-check
√ [kubernetes] control plane can talk to Kubernetes
√ [prometheus] control plane can talk to Prometheus
√ tap api service is running

linkerd-version
---------------
√ can determine the latest version
√ cli is up-to-date

control-plane-version
---------------------
√ control plane is up-to-date
√ control plane and cli versions match

linkerd-prometheus
------------------
√ prometheus add-on service account exists
√ prometheus add-on config map exists
√ prometheus pod is running

linkerd-grafana
---------------
√ grafana add-on service account exists
√ grafana add-on config map exists
√ grafana pod is running

```

### Environment

- Kubernetes Version: 1.16.15
- Cluster Environment: GKE
- Host OS: 
- Linkerd version: stable-2.9.4

**Comment by samstride** [comment]
@guimap , Hi, I'm on 1.19.7-gke.1500 and I don't have this issue.

Wondering if you did a fresh install of 2.9.4 or was it an upgrade to 2.9.4?

Also, did you use the CLI or Helm to install/upgrade?

**Comment by oleh-ozimok** [comment]
Error is reproduced on Chrome Version 88.0.4324.150 (Official Build) (x86_64). Everything works well on safari.

`main.4c9e27fff3e3bb634814.js:24407 TypeError: Failed to execute 'px' on 'CSS': The provided double value is non-finite.
    at rt (main.4c9e27fff3e3bb634814.js:2723)
    at Array.onProcessStyle (main.4c9e27fff3e3bb634814.js:2723)
    at e.t.onProcessStyle (main.4c9e27fff3e3bb634814.js:1)
    at e.t.onProcessRule (main.4c9e27fff3e3bb634814.js:1)
    at Array.forEach (<anonymous>)
    at e.t.process (main.4c9e27fff3e3bb634814.js:1)
    at new e (main.4c9e27fff3e3bb634814.js:1)
    at e.t.createStyleSheet (main.4c9e27fff3e3bb634814.js:1)
    at vn (main.4c9e27fff3e3bb634814.js:2723)
    at main.4c9e27fff3e3bb634814.js:2723
el @ main.4c9e27fff3e3bb634814.js:24407
n.callback @ main.4c9e27fff3e3bb634814.js:24407
da @ main.4c9e27fff3e3bb634814.js:24407
al @ main.4c9e27fff3e3bb634814.js:24407
pc @ main.4c9e27fff3e3bb634814.js:24407
t.unstable_runWithPriority @ main.4c9e27fff3e3bb634814.js:24715
qo @ main.4c9e27fff3e3bb634814.js:24407
dc @ main.4c9e27fff3e3bb634814.js:24407
Jl @ main.4c9e27fff3e3bb634814.js:24407
(anonymous) @ main.4c9e27fff3e3bb634814.js:24407
t.unstable_runWithPriority @ main.4c9e27fff3e3bb634814.js:24715
qo @ main.4c9e27fff3e3bb634814.js:24407
Go @ main.4c9e27fff3e3bb634814.js:24407
$o @ main.4c9e27fff3e3bb634814.js:24407
Kl @ main.4c9e27fff3e3bb634814.js:24407
hi @ main.4c9e27fff3e3bb634814.js:24407
n @ main.4c9e27fff3e3bb634814.js:24715
main.4c9e27fff3e3bb634814.js:2723 Uncaught TypeError: Failed to execute 'px' on 'CSS': The provided double value is non-finite.
    at rt (main.4c9e27fff3e3bb634814.js:2723)
    at Array.onProcessStyle (main.4c9e27fff3e3bb634814.js:2723)
    at e.t.onProcessStyle (main.4c9e27fff3e3bb634814.js:1)
    at e.t.onProcessRule (main.4c9e27fff3e3bb634814.js:1)
    at Array.forEach (<anonymous>)
    at e.t.process (main.4c9e27fff3e3bb634814.js:1)
    at new e (main.4c9e27fff3e3bb634814.js:1)
    at e.t.createStyleSheet (main.4c9e27fff3e3bb634814.js:1)
    at vn (main.4c9e27fff3e3bb634814.js:2723)
    at main.4c9e27fff3e3bb634814.js:2723`

**Comment by adleong** [comment]
Duplicate of https://github.com/linkerd/linkerd2/issues/5612

This was fixed in edge-21.2.3 and the fix is available in stable-2.10.0
